### ISSUE TITLE
iOS opening PDF crashes app with NSInternalInconsistencyException

### ISSUE THREAD

**Issue by mibzman** [sitewaerts/cordova-plugin-document-viewer#140]
```
*** Assertion failure in -[UIApplication _createStatusBarWithRequestedStyle:orientation:hidden:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKitCore_Sim/UIKit-3899.13.13/UIApplication.m:5240
2019-09-26 13:19:36.547090-0400 MyApp[46948:1478202] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'App called -statusBar or -statusBarWindow on UIApplication: this code must be changed as there's no longer a status bar or status bar window. Use the statusBarManager object on the window scene instead.'
```

I've reproduced the issue at [this repo](https://github.com/mibzman/ionic-pdf-crash)

I'm using ionic 5.4.1 with cordova ios 5.0.0.  I can provide version numbers for anything else if needed.

**Comment by tienphan1096** [comment]
I'm running into this issue too, and attempted to fix it in this PR https://github.com/sitewaerts/cordova-plugin-document-viewer/pull/141

**Comment by VicM** [comment]
> I'm running into this issue too, and attempted to fix it in this PR #141

The @tienphan1096 fix have worked for me so far. This is a true blessed fix.

**Comment by regnete** [comment]
thx a lot for your support. just released 0.9.12.
